### DISCUSSION TITLE
Document transparency support

### DISCUSSION THREAD

**Discussion by girlpunk** [QuestPDF/QuestPDF#101] [Ideas]
Not sure if it's something that I've missed, but may be worth mentioning in the doucmentation that the various colour options support ARGB hex codes for transparency?

**Comment by MarcinZiabek** [comment]
Great idea! Do you have any propositions where it is the best place to put this information? Maybe Pattern and Practices? It seems like we have dozens of places mentioning colors. Repeating this information is not viable in this documentation iteration.

**Comment by girlpunk** [comment]
There's already references to the Material Design colors in the patterns section, so that would make sense.

**Comment by MarcinZiabek** [comment]
I have added a small note regarding alpha support [to the documentation](https://www.questpdf.com/documentation/patterns-and-practices.html#color-definitions).

**Comment by girlpunk** [comment]
That looks great, thanks!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
